src/components/MazeContent/MazeContentParts/MazeCard.jsx

```diff
@@ -0,0 +1,25 @@
+import "./../MazeContent.scss";
+import { useMazeStore } from "../../../stores/useMazeStore";
+import { useState } from "react";
+import { MazeDescription } from "./MazeDescription";
+
+export const MazeCard = () => {
+  const [toggleDescription, setToggleDescription] = useState(false);
+  const { description } = useMazeStore();
+
+  // Toggle the show description button.
+  const handleToggleBtn = () => {
+    // Get alla descriptionContainer
+    setToggleDescription(!toggleDescription);
+  };
```

Comment:
I like this idea! On the end screen it would be nice if it was always showing the description